From c1fa677618bd08f8dd7a30652967cd8318e33aa7 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Mon, 5 May 2014 13:31:05 +1000
Subject: [PATCH synaptics 3/3] eventcomm: define the EVIOCGPROP ioctl if the
 kernel doesn't provide it

Saves us rebuilding the driver when the ioctl becomes available in the next
kernel.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 src/eventcomm.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/eventcomm.c b/src/eventcomm.c
index 9990688..c9c8347 100644
--- a/src/eventcomm.c
+++ b/src/eventcomm.c
@@ -44,6 +44,10 @@
 #include <xf86.h>
 #include <mtdev.h>
 
+#ifndef EVIOCGPROP
+#define EVIOCGPROP(len)         _IOC(_IOC_READ, 'E', 0x09, len)
+#endif
+
 #ifndef INPUT_PROP_BUTTONPAD
 #define INPUT_PROP_BUTTONPAD 0x02
 #endif
-- 
1.9.0

